src/basho_bench_worker.erl

```diff
@@ -366,6 +368,14 @@ rate_worker_run_loop(State, Lambda) ->
             exit(ExitReason)
     end.
 
+maybe_report_stats(Op, Result, Stat, State) ->
+    case State#state.should_report_stats of
+        true ->
+            baso_bech_stats:op_complete(Op, Result, Stat);
```

Comment:
baso_bech_stats ?? 
Never accept any cheep imitations ! Always look for the the offical tradmark ...